### DISCUSSION TITLE
Error when install shopify remix shopify app hosted on vercel

### DISCUSSION THREAD

**Discussion by javin1102** [vercel/community#5547] [Help]
### Summary
My app is not much difference from the shopify remix template

### Example

https://tutorialapp-3.vercel.app

### Steps to Reproduce

When i install the app, it gives me an error page stating Serverless Function Error.

![image](https://github.com/vercel/community/assets/61945507/a945ac85-e055-4c86-9dd5-2ff3b47c95e7)
![image](https://github.com/vercel/community/assets/61945507/f445abce-d3c4-4710-8e10-6990f7a7d0a2)

Everything works fine when i install locally

DB: Prisma, Supabase.

**Comment by leerob** [comment]
Looks like it's something to do with your application code: the error message mentions something about an issue with a readable stream. Have you tried making a minimal example?

**Comment by devaublanc** [comment]
Same problem here I tried to replace my  @remix-run/node by @vercel/remix and updated the `entry.server.ts` to steam http response like this https://vercel.com/docs/frameworks/remix#streaming but same problem. I have this random error on hard refresh my page 

```
Unhandled Promise Rejection 	{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"Error: Prisma session table does not exist. 
```

**Reply by andy-pgj** [reply]
I'm having the same issue

**Reply by taxsta** [reply]
ugh, not an enjoyable experience working with remix and shopify apps and hosting i feel like

**Comment by james0r** [comment]
Did you figure it out? Getting the same. Looks like maybe your example URL is working though?
